tests/debugger/do_variables.expected

```diff
@@ -0,0 +1,122 @@
+Connecting to: localhost:5001
+Stopped at tests/debugger/do_variables.js:15
+(jerry-debugger) scopes
+level | type   
+0     | global 
+(jerry-debugger) variables
+name   | type      | value                        
+f      | Function  | function(){/* ecmascript */} 
+addX   | Function  | function(){/* ecmascript */} 
+z      | undefined | undefined                    
+c      | undefined | undefined                    
+print  | Function  | function(){/* ecmascript */} 
+gc     | Function  | function(){/* ecmascript */} 
+assert | Function  | function(){/* ecmascript */} 
+(jerry-debugger) variables 1
+name | type | value 
+(jerry-debugger) variables 0
+name   | type      | value                        
+f      | Function  | function(){/* ecmascript */} 
+addX   | Function  | function(){/* ecmascript */} 
+z      | undefined | undefined                    
+c      | undefined | undefined                    
+print  | Function  | function(){/* ecmascript */} 
+gc     | Function  | function(){/* ecmascript */} 
+assert | Function  | function(){/* ecmascript */} 
+(jerry-debugger) b tests/debugger/do_variables.js:20
+Breakpoint 1 at tests/debugger/do_variables.js:20 (in function() at line:19, col:10)
+(jerry-debugger) c
+Stopped at breakpoint:1 tests/debugger/do_variables.js:20 (in function() at line:19, col:10)
+(jerry-debugger) scopes
+level | type    
+0     | local   
+1     | closure 
+2     | global  
+(jerry-debugger) variables 0
+name | type      | value     
+n    | Number    | 9         
+b    | undefined | undefined 
+(jerry-debugger) variables 1
+name | type   | value 
+x    | Number | 3     
+(jerry-debugger) variables 2
+name     | type     | value                        
+addThree | Function | function(){/* ecmascript */} 
+f        | Function | function(){/* ecmascript */} 
+addX     | Function | function(){/* ecmascript */} 
+z        | Number   | 5                            
+c        | Number   | 4                            
+print    | Function | function(){/* ecmascript */} 
+gc       | Function | function(){/* ecmascript */} 
+assert   | Function | function(){/* ecmascript */} 
+(jerry-debugger) b tests/debugger/do_variables.js:30
+Breakpoint 2 at tests/debugger/do_variables.js:30 (in f() at line:28, col:1)
+(jerry-debugger) c
+Stopped at breakpoint:2 tests/debugger/do_variables.js:30 (in f() at line:28, col:1)
+(jerry-debugger) scopes
+level | type   
+0     | local  
+1     | global 
+(jerry-debugger) variables 1
+name     | type     | value                        
+d        | Number   | 12                           
+addThree | Function | function(){/* ecmascript */} 
+f        | Function | function(){/* ecmascript */} 
+addX     | Function | function(){/* ecmascript */} 
```

Comment:
We should send no values for functions.

Reply:
You are right, it makes sense. I've updated it accordingly.